everglades_workflow.sh

```diff
@@ -3,9 +3,9 @@
 #SBATCH --mail-user=henrysenyondo@ufl.edu 
 #SBATCH --mail-type=FAIL
 #SBATCH --gpus=a100:4
-#SBATCH --cpus-per-task=10
```

Comment:
Is there a reason we're making this number smaller?

Reply:
I think PyTorch has some issues with the number of workers. Ben suggested reducing that number. I can give it a test after our current job finishes today.